app/src/main/java/com/example/playlistmaker/settings/ui/SettingsViewModel.kt

```diff
@@ -50,7 +45,10 @@ class SettingsViewModel (private val context: Context): ViewModel() {
     fun support() {
         val supportIntent = Intent(Intent.ACTION_SENDTO)
         supportIntent.data = Uri.parse("mailto:")
-        supportIntent.putExtra("EXTRA_TEXT", context.getString(R.string.ref_yandex_practicum))
+        supportIntent.putExtra(Intent.EXTRA_EMAIL, context.getString(R.string.share_email))
```

Comment:
В EXTRA_EMAIL нужно передавать массив, а не просто строку, поправь пожалуйста:
```suggestion
        supportIntent.putExtra(Intent.EXTRA_EMAIL, arrayOf(context.getString(R.string.share_email)))
```